locale/en/foreman_remote_execution.po

```diff
@@ -1,19 +1,760 @@
-# foreman_remote_execution
+# English translations for foreman_remote_execution package.
+# Copyright (C) 2016 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the foreman_remote_execution package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
 #
-# This file is distributed under the same license as foreman_remote_execution.
-#
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: version 0.0.1\n"
+"Project-Id-Version: foreman_remote_execution 1.0.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2014-08-20 08:46+0100\n"
-"PO-Revision-Date: 2014-08-20 08:54+0100\n"
-"Last-Translator: Foreman Team <foreman-dev@googlegroups.com>\n"
-"Language-Team: Foreman Team <foreman-dev@googlegroups.com>\n"
-"Language: \n"
+"PO-Revision-Date: 2016-02-03 04:48-0500\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: English\n"
+"Language: en\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"\n"
+
+msgid ""
+"Another interface is already set as execution. Are you sure you want to use th"
+"is one instead?"
+msgstr ""
+
+msgid ""
+"<div class=\"line error\">There was an error while updating the status, try <a h"
+"ref=\"javascript:window.location.href=window.location.href\">refreshing</a> the "
```

Comment:
You should extract the link out of here, which is particularly tricky without fragmenting the sentence, but the translation shouldn't contain JS.